src/test/java/com/revature/services/JMeterServiceTest.java

```diff
@@ -244,7 +253,59 @@ void testCreateLoopController() {
     void testCreateLoopControllerNull() {
         assertNull(jm.createLoopController(null, loadConfig.getLoops()));
     }
+    
+    // ---------------- Body Tests -------------------
+    @Test
+    void testObjectCreatorSimplePost() {
+        testSpecs = new SwaggerDocutest(adapter.getRequests(TestUtil.post));
+        Set<HTTPSamplerProxy> samplers = jm.createHTTPSamplerProxy(testSpecs);
+        
+        for (HTTPSamplerProxy element : samplers) {
+            Arguments args = element.getArguments();
+            Map<String, String> argsMap = args.getArgumentsAsMap();
+            String bodyText = argsMap.get("").replaceAll("\\s", "");     
+            String expected = TestUtil.POST_OBJ_JSON.replaceAll("\\s", "");
+
+            if (!argsMap.isEmpty()) {
+                assertEquals(expected, bodyText);
+            } else {
+                assertNull(bodyText);
+            }
+        }
 
+    }
+    
+    @Test
+    void testObjectCreatorHTTPSamplerProxy() {
+        testSpecs = new SwaggerDocutest(adapter.getRequests(TestUtil.petstore));
+        Set<HTTPSamplerProxy> samplers = jm.createHTTPSamplerProxy(testSpecs);
+        for (HTTPSamplerProxy element : samplers) {
+            Arguments args = element.getArguments();
+            Map<String, String> argsMap = args.getArgumentsAsMap();
+            String bodyText = argsMap.get("");            
+
+            if (!argsMap.isEmpty()) {
+                System.out.println(bodyText);
+                // argsMap returns a string with "null" if no valid values
+                assertNotEquals("null",bodyText);
+            } else {
+                assertNull(bodyText);
+            }
+        }
+    }
+    
+    @Disabled
+    @Test
+    void sanityCheck() {
```

Comment:
I understand the need for this while developing, but this just means we will need to make sure we write up clear enough documentation for the following iterations on this project to quickly get up to speed.